src/app/feedback/feedback-view.component.ts

```diff
@@ -22,7 +22,7 @@ export class FeedbackViewComponent implements OnInit, OnDestroy {
   user: any = {};
   newMessage = '';
   isActive = true;
-  editTitleMode = false;
+  toggleTitleMode = false;
```

Comment:
Sorry to ask for a reversion here, but I think I just missed the clear way to name these.  This variable should remain `editTitleMode` and the method should change to `toggleEditTitleMode`

Reply:
Actually with a little more thought I realize your suggestion `changeEditTitleMode` would be better since we supply a value to the function.